android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -675,6 +676,18 @@ private void pushDeactivate(@NonNull MethodCall call, @NonNull MethodChannel.Res
     }
   }
 
+  private void pushReset(@NonNull MethodCall call, @NonNull MethodChannel.Result result) {
+    final AblyFlutterMessage message = (AblyFlutterMessage) call.arguments;
+    try {
+      Integer handle = (Integer) message.message;
```

Comment:
The names are quite vague :sweat_smile: And the `message.message` could be improved :wink: Could you think of some better names? :thinking:

Reply:
I'm aware of that, and already working on it in scope of #179, but it's a large change, so for now I decided to stick to previous convention which missmatches `handle` and `message` values in multiple cases 😞